### ISSUE TITLE
Webcam failed to start in Chrome

### ISSUE THREAD

**Issue by ara4n** [vector-im/matrix-video-chat#129]
@antpb tried to join a matrixvoip.dev URL from Chrome and got:

"Access to fetch at 'https://sentry.matrix.org/api/41/envelope/?sentry_key=...&sentry_version=7…' from origin 'https://matrixvoip.dev' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
http://sentry.matrix.org/api/41/envelope/?sentry_key=...&sentry_version=7:1… Failed to load resource: net::ERR_FAILED
index.2532949d.js:1 Uncaught (in promise) DOMException: Could not start video source"

**Comment by robertlong** [comment]
Closing as duplicate of #95
